### DISCUSSION TITLE
Improvement on arrows and specifically curve arrows

### DISCUSSION THREAD

**Discussion by hamidhaghdoost** [jgraph/drawio#3832] [General]
Hi. I am really satisfied with this software and I appreciate you for such a great software for free. 

There is a problem that I don't really know whether I can't use it or it is everybody's problem. 

While using arrows specifically curve arrow, it is really difficult to adjust it properly such that either looks well and be in the precise place. 

I was trying Miro which is a similar software and it gives you a kind of arrow that you can move object and arrow is being adjusted automatically. 

I know that the one in draw.io has much more flexibility and you can adjust it how ever that you want but most of the time, we just need a curve arrow from one to another with a reasonable amount of smoothness so it seems that having sth like Miro would be more applicable. 

Thank you

**Comment by davidjgraph** [comment]
Yes, Miro does sound like the better choice for you.
